src/ducks/recurrence/search-with-existing-recurrence.spec.js

```diff
@@ -842,4 +843,56 @@ describe('recurrence scenario with multiple amounts and categories', () => {
       '\n' + sortBy(updatedRecurrences.map(formatRecurrence)).join('\n')
     ).toMatchSnapshot()
   })
+
+  it('should update existing bundle, even for more than 100 days back', () => {
+    const oldNetflixTransactions = [
+      {
+        _id: 'january',
```

Comment:
nit: it would be nice to start optimizing the file size ^^ like that for example :
```jsx
const baseTransaction = {
        amount: -50,
        label: 'Netflix',
        automaticCategoryId: 'NetflixId',
        localCategoryId: '0',
        account: 'accountId'
}
const oldNetflixTransactions = [
      {
        _id: 'january',
        date: '2021-01-01T12:00:00.000Z',
        ...baseTransaction
      },
     {
        _id: 'february',
        date: '2021-02-01T12:00:00.000Z',
        ...baseTransaction
      },
      ...
]
```

Reply:
Exact ! J'ai encore des sujets sur la récurrence, je le passerai dans une autre PR